dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/runner/MasterSchedulerService.java

```diff
@@ -273,7 +273,7 @@ private void scheduleProcess() throws Exception {
 
     private List<Command> findCommands() {
         int pageNumber = 0;
-        int pageSize = masterConfig.getFetchCommandNum();
+        int pageSize = masterConfig.getFetchCommandNum() * ServerNodeManager.MASTER_SIZE;
```

Comment:
If have many master server, fetch size will be very large, I'm not sure whether it's a hidden danger.

Reply:
let have a back view to the core schedule 2.0, which main point is that :

1, parallel the schedule when we have many master, and to avoid conflict when do delete transation among several master, so we use slot which is get from registry of zk,
so we can make sure every master can handle the command which is belong to it, am i wright ?


2, and also i assume that chage the single commnand to command list is that we belive the master can handle the several command, am i wright ?

@caishunfeng  maybe we shodld make a unify idea for above

Reply:
@Yao-MR Yes, that's the point. Sorry for the late reply.